helm/sematic-server/templates/deployment.yaml

```diff
@@ -1,16 +1,30 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
+  {{- if .Values.deployment.socket_io.dedicated }}
+  name: {{ .Release.Name }}-api
```

Comment:
I think the name of the service should be kept consistent regardless whether it's accompanied by another, "socketio" service.

Reply:
+1

Reply:
👍